TRANSLATION.md

```diff
@@ -0,0 +1,140 @@
+Translating Kibana
```

Comment:
I'll hold back my review of the docs until we move them there, since the transition will wipe out all comments anyway.